tests/test_universe.py

```diff
@@ -300,6 +303,11 @@ def test_complex_transform_fill_init(self):
         assert not fill.hidden_transform
         assert fill.old_universe_number == 5
         assert fill.old_transform_number == 3
+        # test sparse fill
+        cell = Cell("1 0 -1 fill=0:0 0:1 0:1 1 1 1")
+        fill = cell.fill
+        assert fill.old_universe_numbers[0, 0, 0] == 1
+        assert fill.old_universe_numbers[0, 1, 1] == 0
```

Comment:
I'd like to see another sparse fill test with numbers other than 0 or 1, and constructed such that we can verify that y/x aren't inverted. I've seen that way too many times